pkg/services/sqlstore/org.go

```diff
@@ -214,3 +217,60 @@ func DeleteOrg(cmd *m.DeleteOrgCommand) error {
 		return nil
 	})
 }
+
+func getOrCreateOrg(sess *DBSession, orgName string) (int64, error) {
+	var org m.Org
+
+	if setting.AutoAssignOrg {
+		has, err := sess.Where("id=?", setting.AutoAssignOrgId).Get(&org)
+		if err != nil {
+			return 0, err
+		}
+		if has {
+			return org.Id, nil
+		}
+		if setting.AutoAssignOrgId == 1 {
+			org.Name = "Main Org."
```

Comment:
should this be orgName?

Reply:
In this PR I would say no, it currently works this way in master. Need to look into this later.